data_collection/gazette/spiders/ce_juazeiro_do_norte.py

```diff
@@ -0,0 +1,62 @@
+import datetime as dt
+
+import scrapy
+import re
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class CeJuazeiroDoNorteSpider(BaseGazetteSpider):
+    TERRITORY_ID = "2307304"
+    name = "ce_juazeiro_do_norte"
+    allowed_domains = ["juazeiro.ce.gov.br"]
+    start_date = None
```

Comment:
**suggestion**: Provide a default value for this field. It will be replaced by the value of `start_date` argument if we provide it when running the spider.

`start_date = dt.date(2009, 1, 1)`

Reply:
Changed on https://github.com/okfn-brasil/querido-diario/pull/358#discussion_r520138351 :+1: